### DISCUSSION TITLE
Renaming of assets preserving history

### DISCUSSION THREAD

**Discussion by the4thamigo-uk** [dagster-io/dagster#19156] [Q&A] [answered]
Is it possible to rename an asset *or change its asset key prefix) and have its materialization history preserved?

**Comment by OwenKephart** [comment, marked as answer]
Hi @the4thamigo-uk , unfortunately, this is not currently possible: https://github.com/dagster-io/dagster/discussions/17676

**Comment by the4thamigo-uk** [comment]
Ah I dint spot that... good to see you are already considering this. Thanks.
